res/css/_common.pcss

```diff
@@ -56,8 +56,10 @@ limitations under the License.
     /* This is required to ensure Compound tooltips correctly draw where they should with z-index: auto */
     contain: strict;
 }
-.mx_Dialog_StaticContainer,
-.mx_Dialog_Container {
+#mx_ContextualMenu_Container,
+#mx_PersistedElement_container,
+#mx_Dialog_Container,
+#mx_Dialog_StaticContainer {
```

Comment:
why are these changing?

Reply:
So that they layer correctly relative to the PIP widget and tooltips and each other by having their own CSS stacking contexts